docs/guides/endgame.md

```diff
@@ -757,7 +758,7 @@ following steps to release:
 ### Release Candidate releases
```

Comment:
Is this a proper noun? I don't think it is.

Reply:
```suggestion
### Release candidate releases
```